src/CLI/Resources/OptionsHelpText.resx

```diff
@@ -135,6 +135,9 @@
   <data name="ScanId" xml:space="preserve">
     <value>Scan ID</value>
   </data>
+  <data name="ScanMultipleWindows" xml:space="preserve">
+    <value>Determines if all windows should be scanned (true) or not (false).</value>
```

Comment:
This text feels confusing. We're not scanning all windows, since we've scoped to a single process. Maybe something like "Determines if multiple top-level windows of the target process (true) or just the first one (false)."

Reply:
Very good point, I've switched to your wording now and updated the parameter name everywhere applicable.